docs/user-guides/thanos_querier.md

```diff
@@ -0,0 +1,62 @@
+# Using ThanosQuerier to federate MonitoringStacks
+
+`ThanosQuerier` can be used to query data from a set of `MonitoringStack` resources.
+
+## Architecture
+
+A `ThanosQuerier` resource selects a set of `MonitoringStack` resources using
+label and namespace selectors.
+
+Under the hood, the observability operator creates a Kubernetes Deployment
+which is configured to connect to the Thanos sidecars running in the Prometheus
+pods.
+
+## Tutorial
+
+### Pre-requisites
+* Observability operator installed and running.
+* Cluster admin permissions.
+
+### Installation
+
+We are going to create
+* 1 `MonitoringStack` running in the `project-a` namespace.
+* 1 `MonitoringStack` running in the `project-b` namespace.
+* 1 `ThanosQuerier` running in the `project-c` namespace.
+
```

Comment:
how about adding a diagram in a way such as

```
┌─────────────────┐    ┌─────────────────┐    ┌─────────────────┐
│   project-a     │    │   project-b     │    │   project-c     │
│ MonitoringStack │    │ MonitoringStack │    │ ThanosQuerier   │
│                 │    │                 │    │                 │
│ ┌─────────────┐ │    │ ┌─────────────┐ │    │ ┌─────────────┐ │
│ │ Prometheus  │ │    │ │ Prometheus  │ │    │ │ Thanos      │ │
│ │ + Thanos    │ │◄───┤ │ + Thanos    │ │◄───┤ │ Query       │ │
│ │ Sidecar     │ │    │ │ Sidecar     │ │    │ │             │ │
│ └─────────────┘ │    │ └─────────────┘ │    │ └─────────────┘ │
└─────────────────┘    └─────────────────┘    └─────────────────┘

Reply:
@danielmellado in fact the diagram is thanos querier talking to both monitoringstacks